registration/registration-services/pom.xml

```diff
@@ -382,7 +382,7 @@
 		<dependency>
 			<artifactId>kernel-biosdk-provider</artifactId>
 			<groupId>io.mosip.kernel</groupId>
-			<version>${mosip.core.kernel.version}</version>
+			<version>${mosip.bio-utils.version}</version>
 			<exclusions>
 				<exclusion>
 					<groupId>io.mosip.kernel</groupId>
```

Comment:
1. If these are not getting used then we can remove this or we can refer from parent pom:  https://github.com/mosip/registration-client/blob/release-1.3.x/registration/registration-api/pom.xml#L20 https://github.com/mosip/registration-client/blob/release-1.3.x/registration/registration-api-stub-impl/pom.xml#L21

Reply:
removed unused and repeated properties in child pom.